src/Fees.sol

```diff
@@ -32,24 +32,29 @@ abstract contract Fees is IFees, Owned {
         controllerGasLimit = _controllerGasLimit;
     }
 
-    function _fetchProtocolFees(PoolKey memory key) internal view returns (uint24 protocolFees) {
-        uint16 protocolSwapFee;
-        uint16 protocolWithdrawFee;
+    /// @notice Fetch the protocol fees for a given pool, returning false if the call fails or the returned fees are invalid.
+    /// @dev the success of this function must be checked when called in setProtocolFees
```

Comment:
Could you add comment that we DONT check the success value in initialize because even if the call is unsuccessful we need to allow pools to initialize. New pools will just default initialize then to 0 protocolFee